### PR TITLE
feature/content loader new

### PR DESCRIPTION
**Pull request by haideralishah**
Content loader Implement.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -78,12 +78,6 @@ project.ext.react = [
 
 apply from: "../../node_modules/react-native/react.gradle"
 
-project.ext.vectoricons = [
-        iconFontNames: ['FontAwesome.ttf', 'FontAwesome5_Brands.ttf', 'FontAwesome5_Regular.ttf', 'FontAwesome5_Solid.ttf',
-                        'MaterialIcons.ttf'] // Name of the font files you want to copy
-]
-apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
-
```
Comment (norbertschuler) [inline android/app/build.gradle]: Why have you removed the module `react-native-vector-icons` again? Is this a merge error as you have not removed it from the iOS project? This questions affects your changes in `android/app/build.gradle` and  `android/app/src/main/java/org/pftp/MainApplication.java`.

File: android/build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:3.3.1'
```
Comment (norbertschuler) [inline android/build.gradle]: Why have you downgraded the latest gradle version here?

File: app/config/index.js.dist [hunk 3]
```diff
@@ -1,26 +0,0 @@
...
-  currency: 'EUR',
-  mapIds: { inventory: 'dee6acf9de774fe6878813f707b4ab88'},
-  bugsnagApiKey: '',
-  android:{
-    appId: 'org.pftp'
-  },
-  ios:{
-    appId: '1444740626'
-  }
-};
```
Comment (norbertschuler) [inline app/config/index.js.dist]: Please do not delete our index.js.dist file. We need it for automatic deployment.

File: package.json [hunk 4]
```diff
@@ -44,6 +44,7 @@
     "jwt-decode": "^2.2.0",
     "liform-react": "^0.9.0",
     "lodash": "^4.17.10",
+    "node": "^10.17.0",
```
Comment (norbertschuler) [inline package.json]: Where does this package get used?

File: package.json [hunk 5]
```diff
@@ -107,6 +109,7 @@
     "reselect": "^4.0.0",
     "rxjs": "^5.5.6",
     "scheduler": "^0.17.0",
+    "sharp": "^0.23.2",
```
Comment (norbertschuler) [inline package.json]: Where does this package get used? I am just curious to know.

File: package.json (hunk 4 above)
Comment (norbertschuler) [reply package.json]: I would prefer not to install another local node binary as every developer should use his own one.

**Comment by sagararyal** [comment]
@haideralishah doesn't build for me anymore.

File: android/app/build.gradle (hunk 1 above)
Comment (haideralishah) [reply android/app/build.gradle]: I re-add this module and configuration.

File: android/build.gradle (hunk 2 above)
Comment (haideralishah) [reply android/build.gradle]: Gradle version is Fixed now.

File: app/config/index.js.dist (hunk 3 above)
Comment (haideralishah) [reply app/config/index.js.dist]: Fixed

File: package.json (hunk 4 above)
Comment (haideralishah) [reply package.json]: I removed this module.

File: package.json (hunk 5 above)
Comment (haideralishah) [reply package.json]: I removed this module.

File: android/app/build.gradle (hunk 1 above)
Comment (norbertschuler) [reply android/app/build.gradle]: Right, but you have not reverted the deletions in `android/app/build.gradle` and `android/app/src/main/java/org/pftp/MainApplication.java`. Please do that as well.

Comment (haideralishah) [reply android/app/build.gradle]: Fixed.

Comment (haideralishah) [reply android/app/build.gradle]: I rename the branch now its `feature/content loader new`.
Sorry for it, it was a little typo mistake.

**Comment by norbertschuler** [comment]
> @haideralishah doesn't build for me anymore.

I also can not build it. Please try creating a new pull request with the renamed `feature/content_loader_new` branch and merge our latest develop branch and solve the merge conflicts.

**Comment by norbertschuler** [comment]
replaced with #1715

**Pull request closed without merging**